core_lib/src/structure/filemanager.cpp

```diff
@@ -255,8 +271,8 @@ Status FileManager::save(const Object* object, const QString& sFileName)
     QString sMainXMLFile;
     QString sDataFolder;
 
-    const bool isOldType = sFileName.endsWith(PFF_OLD_EXTENSION);
-    if (isOldType)
+    Status isArchiveSt = isArchiveFormat(sFileName);
+    if (isArchiveSt == Status::NOT_ARCHIVE_FORMAT)
```

Comment:
You're right.. previously I was using the sanity check function, and used the Status details to enrich the error report but since we don't do that anymore, there's no need to use Status, I will change that

Reply:
fixed